src/moscot/base/output.py

```diff
@@ -168,6 +171,98 @@ def chain(self, outputs: Iterable["BaseSolverOutput"], scale_by_marginals: bool
 
         return op
 
+    def sparsify(
+        self,
+        mode: Literal["threshold", "percentile", "min_row"],
+        value: Optional[float] = None,
+        batch_size: int = 1024,
+        n_samples: Optional[int] = None,
+        seed: Optional[int] = None,
+    ) -> "MatrixSolverOutput":
+        """
+        Sparsify the :attr:`transport_matrix`.
+
+        This function sets all entries of the transport matrix below `threshold` to 0 and
+        returns an instance of :class:`moscot.base.MatrixSolverOutput` with the the
```

Comment:
I wouldn't link to the `MatrixSolverOutput`, it's already a in the type hint (not to mention this link isn't correct)>

Reply:
Do you mean completely dropping the piece of information that we return an instance of `MatrixSolverOutput`? That's difficult I think because it's useful to describe that we save the `transport_matrix` as a `csr_matrix`. For now, I just remove the link.

Reply:
Only the former is good.

Reply:
not sure if I understand what you mean , but please have a look at the changes.